src/aat/resources/features/F-065/F-065.feature

```diff
@@ -68,3 +68,34 @@
       Then a negative response is received
       And the response [has an HTTP-401 Unauthorized]
       And the response has all other details as expected
+
+    @S-580
+    Scenario:Must return the case view with only the respective fields with READ access for Solicitors
+      Given a user with [an active Solicitor profile in CCD with a specific variation of ACLs on a case type]
+      And a user with [another active solicitor profile with different variation of ACLs on the same case type]
+      And a successful call [to create a token for case creation] as in [Befta_Jurisdiction2_Default_Token_Creation_Data_For_Case_Creation]
+      And another successful call [by a privileged user with full ACL to create a case of this case type] as in [Befta_Jurisdiction2_Default_Full_Case_Creation_Data]
+      When a request is prepared with appropriate values
+      And the request [is prepared on behalf of the privileged user creating the case above]
+      And it is submitted to call the [Retrieve a case by ID for dynamic display] operation of [CCD Data Store]
```

Comment:
change Retrieve to retrieve

Reply:
I can't change it. As, it define in Controller logic by CCD Development team.